buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -23,6 +23,7 @@ object Dependencies {
   val kotlinGradlePlugin = "org.jetbrains.kotlin:kotlin-gradle-plugin:${baseKotlin}"
   val kotlinReflect = "org.jetbrains.kotlin:kotlin-reflect"
   val kotlinSerialization = "org.jetbrains.kotlin:kotlin-serialization:${baseKotlin}"
+  val kotlinTest = "org.jetbrains.kotlin:kotlin-test:1.5.31"
```

Comment:
```suggestion
  val kotlinTest = "org.jetbrains.kotlin:kotlin-test"
```
You'll get the right version automatically

Reply:
Fixed.